test/llilc_checkpass.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python
+#
+#title           :llilc_checkpass.py
+#description     :
+# 
+# llilc_checkpass compares two test rusuts, no matter summary or verbose,
+# checks if any function successfully jitted by LLILC in base result
+# failed jitting in diff result.
+#
+# If the check passed, the script return 0. In case of 0, it might have
+# jitted more functions in diff result than in base result. The two results
+# are not necessary the same in this case. Counting result is reported.
+# 
+# If the check failed due to newly faily routine, return 1.
+#     
+# If the check failed due to unexpected reason, return negative numbers.
+#
+# usage: llilc_checkpass.py [-h] -b BASE_RESULT_PATH -d DIFF_RESULT_PATH
+#
+# optional arguments:
+#   -h, --help            show this help message and exit
+#   -b BASE_RESULT_PATH, --base-result-path BASE_RESULT_PATH
```

Comment:
@adiaaida This is still an open Python issue. See http://bugs.python.org/issue9694. The issue is that required arguments are listed under optional arguments section. Not sure if it is fixed in Python 3.5. But in 3.4.2 the problem is still there. But we do not assume developer all use latest Python. After some searching, I think the best solution would be using add_argument_group with title "required arguments". Actually the required argument style was from you in other PR. I used to use positional argument in these cases.

The required arguments style was actually from you in other PR. I used to use positional argument for this kind of arguments.